deployments/scripts/run_eth2sub.sh

```diff
@@ -1,19 +0,0 @@
-#!/bin/bash
```

Comment:
Can we add some reference to https://github.com/svyatonik/parity-bridges-common.test in the README given these scripts are now removed?

Reply:
I'd rather see us develop some simplified test/dev scripts for the repo, and leave the more complicated setups (like some in that repo) as something do be done with Compose.

Reply:
I'm not suggesting to include any of these scripts, just to add a one line in the README to point to this repo since it might be helpful for someone who is not willing to or can't run docker / docker-compose.